pip/commands/completion.py

```diff
@@ -10,32 +11,35 @@
 
 COMPLETION_SCRIPTS = {
     'bash': """
-_pip_completion()
-{
-    COMPREPLY=( $( COMP_WORDS="${COMP_WORDS[*]}" \\
-                   COMP_CWORD=$COMP_CWORD \\
-                   PIP_AUTO_COMPLETE=1 $1 ) )
+        _pip_completion()
```

Comment:
what about putting each script into a resource file and using pkg_resources to get them